Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM openshift/origin-release:golang-1.10 as builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/cluster-image-registry-operator
 COPY . .
 RUN make build
 
-FROM centos:7
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/cluster-image-registry-operator/tmp/_output/bin/cluster-image-registry-operator /usr/bin/
 RUN useradd cluster-image-registry-operator
 USER cluster-image-registry-operator
-COPY --from=builder /go/src/github.com/openshift/cluster-image-registry-operator/tmp/_output/bin/cluster-image-registry-operator /usr/bin
```

Comment:
why did you reorder this?  the previous ordering was optimized for layer cache reuse.

(actually probably the best order would be for the binary copy to be the last statement)

```
LABEL blah
RUN useradd
USER foo
COPY manifest content (slowly changing)
COPY binary (fast changing)
```

is probably the ideal ordering.

Reply:
Three is no cache reuse in any of our build systems anymore.  Also, we are making this consistent across everyone, and people having special logic with ordering is confusing.

For 99% of binaries, we should be able to generate and validate these docker files from a central spot.  That's our end goal, because the variance here doesn't have much benefit.